csharp/src/Ice/OpaqueEndpoint.cs

```diff
@@ -24,6 +25,7 @@ public sealed class OpaqueEndpoint : Endpoint
 
         public override int Timeout => -1;
         public override EndpointType Type { get; }
+        public override string Transport => "";
```

Comment:
Should this be "opaque" like in stringified endpoints?
Or maybe the actual transport when Type is a known type?

Reply:
I removed Transport, we can just use `Name` that already return "opaque"